nntrainer/utils/base_properties.cpp

```diff
@@ -32,6 +32,23 @@ std::string str_converter<str_prop_tag, std::string>::from_string(
   return value;
 }
 
+template <>
+std::string str_converter<bool_prop_tag, bool>::to_string(const bool &value) {
```

Comment:
shouldn't this also throw if value is empty? if a property is empty, we dont want to save it. or this will be handled somewhere else? My concern is if a prop is empty, then it should not be part of saved properties.

Reply:
If empty we throw at the property level. You might wanna double check at `base_properties.h:182`.

Reply:
great, thanks for the check.